src/test/java/seedu/parser/ParserTest.java

```diff
@@ -0,0 +1,180 @@
+package seedu.parser;
+
+import org.junit.jupiter.api.Test;
+import seedu.data.Book;
+import seedu.data.SysLibException;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ParserTest {
+
+    @Test
+    public void testProcessExitCommand() {
+        Parser parser = new Parser();
+        String validResponse = "exit";
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
+
+        String expectedOutput = "Bye, hope to see you again soon!"+ System.lineSeparator()+
+                "____________________________________________________________"+System.lineSeparator();
+        assertEquals(expectedOutput, output);
+    }
+
+    @Test
+    public void testProcessHelpCommand() {
+        Parser parser = new Parser();
+        String validResponse = "help";
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
```

Comment:
Do you want to extract this to a new class like TestUtils then put this code as a method which returns the output, so all the test methods can use this to get the output message. If not this code repeats a lot